releases/v21.1.0-alpha.2.md

```diff
@@ -0,0 +1,544 @@
+- The parameter `buffered-writes` for file sink configurations has been added. It is set to `true` (writes are buffered) by default; and set to `false` (i.e., avoid buffering and flush every log entry) when the `auditable` flag is requested. [#58995][#58995] {% comment %}doc{% endcomment %}
+- The default output format for `file-group` and `stderr` sinks has been changed to `crdb-v2`. This new format is non-ambiguous and makes it possible to reliably parse log files. Refer to the format's [documentation](https://github.com/cockroachdb/cockroach/blob/master/docs/generated/logformats.md) for details. Additionally, it prevents single log lines from exceeding a large size; this problem is inherent to the `crdb-v1` format and can prevent [`cockroach debug zip`](../v21.1/cockroach-debug-zip.html) from retrieving `v1` log files. The new format has also been designed so that existing log file analyzers for the `crdb-v1` format can read entries written the new format. However, this conversion may be imperfect. Refer to the new format's [documentation](https://github.com/cockroachdb/cockroach/blob/master/docs/generated/logformats.md) for details. In case of incompatibility, users can force the previous format by using `format: crdb-v1` in their logging configuration. [#59087][#59087] {% comment %}doc{% endcomment %}
+
+### API endpoint changes
+
+- The notable event `create_statistics` is only reported when the [cluster setting](../v21.1/cluster-settings.html) `sql.stats.post_events.enabled` is enabled. This fact is now also reported in the [event log reference documentation](https://github.com/cockroachdb/cockroach/blob/master/docs/generated/eventlog.md). [#57877][#57877] {% comment %}doc{% endcomment %}
+- The `Timestamp` field of structured notable events is now numeric, and encodes a number of nanoseconds since the unix epoch. [#58070][#58070] {% comment %}doc{% endcomment %}
+- The Health API now checks that the SQL server is ready to accept clients when a readiness check is requested. [#59350][#59350] {% comment %}doc{% endcomment %}
+
+### DB Console changes
+
+- Minor style changes to represent new branding palette. [#57130][#57130]
+- Changed default per-page value on [Transactions page](../v21.1/ui-transactions-page.html) to 20; minor style updates. [#57824][#57824]
+- Added a timeseries graph indicating statement denials due to feature flags on the [SQL Dashboard](../v21.1/ui-sql-dashboard.html) of DB Console. [#57533][#57533] {% comment %}doc{% endcomment %}
+- Updated labels on [Hardware Dashboard](../v21.1/ui-hardware-dashboard.html) to be more accurate. [#57224][#57224]
+- Link back to [Statements page](../v21.1/ui-statements-page.html) from [Statement Details page](../v21.1/ui-statements-page.html#statement-details-page) will always link to the Statements list instead of invoking "back" action on the browser. [#57975][#57975]
```

Comment:
```suggestion
- Updated labels on the [Hardware Dashboard](../v21.1/ui-hardware-dashboard.html) to be more accurate. [#57224][#57224]
- Updated the link back to [Statements](../v21.1/ui-statements-page.html) from [Statement Details](../v21.1/ui-statements-page.html#statement-details-page) so that it will always link to the Statements list instead of invoking the "back" action on the browser. [#57975][#57975]
```